src/components/atoms/Button/Button.test.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import Button from 'components/atoms/Button';
-import { ReactComponent as icon } from 'assets/button-icon.svg';
+import { ButtonIcon } from 'assets/index';
```

Comment:
```suggestion
import { ButtonIcon } from 'assets';
```
index.[js|ts|jsx|tsx] works by default. We can skip it